site/app/views/NavigationView.php

```diff
@@ -710,7 +717,14 @@ private function getGradeButton(Gradeable $gradeable, int $list_section) {
 
             if ($gradeable->getType() === GradeableType::ELECTRONIC_FILE) {
                 if ($gradeable->isTaGrading()) {
-                    $TA_percent = $gradeable->getGradingProgress($this->core->getUser());
+
+                    $cookie_string = "include_bad_submissions__" . $gradeable->getId();
+                    $bad_submissions = array_key_exists($cookie_string, $_COOKIE) ? ($_COOKIE[$cookie_string]==="include" ? true : false) : false;
+
+                    $cookie_string = "include_null_section__" . $gradeable->getId() ;
+                    $null_section = array_key_exists($cookie_string, $_COOKIE) ? ($_COOKIE[$cookie_string]==="include" ? true : false) : false;
```

Comment:
Same here

Reply:
thanks for your suggestion, the code have changed